mpas_analysis/sea_ice/climatology_map_berg_conc.py

```diff
@@ -0,0 +1,232 @@
+    observations
+    """
+    # Authors
+    # -------
+    # Darin Comeau, Xylar Asay-Davis
+
+    def __init__(self, config, mpasClimatologyTask, hemisphere,
+                 refConfig=None):  # {{{
+        """
+        Construct the analysis task.
+
+        Parameters
+        ----------
+        config :  ``MpasAnalysisConfigParser``
+            Configuration options
+
+        mpasClimatologyTask : ``MpasClimatologyTask``
+            The task that produced the climatology to be remapped and plotted
+
+        hemisphere : {'NH', 'SH'}
+            The hemisphere to plot
+
+        refConfig :  ``MpasAnalysisConfigParser``, optional
+            Configuration options for a reference run (if any)
+        """
+        # Authors
+        # -------
+        # Darin Comeau, Xylar Asay-Davis
+
+        taskName = 'climatologyMapIcebergConc{}'.format(hemisphere)
+
+        fieldName = 'IcebergConc'
+        # call the constructor from the base class (AnalysisTask)
+        super(ClimatologyMapIcebergConc, self).__init__(
+                config=config, taskName=taskName,
+                componentName='seaIce',
+                tags=['icebergs', 'climatology', 'horizontalMap', fieldName])
+
+        mpasFieldName = 'timeMonthly_avg_bergAreaCell'
+        iselValues = None
+
+        sectionName = taskName
+
+        if hemisphere == 'NH':
+            hemisphereLong = 'Northern'
+        else:
+            hemisphereLong = 'Southern'
+
+        # read in what seasons we want to plot
+        seasons = config.getExpression(sectionName, 'seasons')
+
+        if len(seasons) == 0:
+            raise ValueError('config section {} does not contain valid list '
+                             'of seasons'.format(sectionName))
+
+        comparisonGridNames = config.getExpression(sectionName,
+                                                   'comparisonGrids')
+
+        if len(comparisonGridNames) == 0:
+            raise ValueError('config section {} does not contain valid list '
+                             'of comparison grids'.format(sectionName))
+
+        # the variable self.mpasFieldName will be added to mpasClimatologyTask
+        # along with the seasons.
+        remapClimatologySubtask = RemapMpasClimatologySubtask(
+            mpasClimatologyTask=mpasClimatologyTask,
+            parentTask=self,
+            climatologyName='{}{}'.format(fieldName, hemisphere),
+            variableList=[mpasFieldName],
+            comparisonGridNames=comparisonGridNames,
+            seasons=seasons,
+            iselValues=iselValues)
+
+        if refConfig is None:
+            refTitleLabel = 'Observations (Altiberg)'
+            galleryName = 'Observations: Altiberg'
+            diffTitleLabel = 'Model - Observations'
+            refFieldName = 'icebergConc'
+            obsFileName = build_config_full_path(
+                    config, 'icebergObservations',
+                    'concentration{}'.format(hemisphere))
```

Comment:
I think this should be:
```python
'concentrationAlteberg{}'.format(hemisphere))
```
because the corresponding entry in `config.default` got renamed.

Reply:
@xylar Thanks for catching this - I just pushed this change.